src/pages/login/login.component.js

```diff
@@ -148,4 +154,4 @@ const SignIn = () => {
   );
 };
 
-export default SignIn;
+export default connect()(SignIn);
```

Comment:
Instead connect store to the components, what do you think about `useSelector` hook?